tests/envs/test_pyperplan_env.py

```diff
@@ -1,36 +1,14 @@
 """Tests for PyperplanEnv()."""
 
 from llm4pddl import utils
+from llm4pddl.envs import PYPERPLAN_BENCHMARKS
```

Comment:
does this line import the Benchmarks correctly even though they are in the __init__ file? are all variable located in the parent directory (in this case `envs`) when defined in the __init__ file?

Reply:
yup! the `__init__.py` file is special in that way